workflows/rnaseq/downstream/rnaseq.Rmd

```diff
@@ -291,6 +293,22 @@ res.list[['lfc2']] <- list(
 )
 ```
 
+```{r assemble_variables, dependson=c('results', 'results2'), cache=TRUE}
```

Comment:
see comment above about naming. Also, if the docs-checking code is working properly, we'll get an error because this doesn't have a corresponding `assemble_variables` entry in the docs.